app/components/header_component/view.rb

```diff
@@ -2,10 +2,10 @@
 
 module HeaderComponent
   class View < ViewComponent::Base
-    def initialize(current_user)
+    def initialize(user)
```

Comment:
It's because `current_user`is also the name of the signon method we are/were using - when we were changing from `current_user` to `@current_user` elsewhere we found it a bit confusing that we were also using `current_user` as the parameter name in this class. You're right though, we should add something about this to the commit message